### DISCUSSION TITLE
Docker compose as sample of all services included by AIO?

### DISCUSSION THREAD

**Discussion by webysther** [nextcloud/all-in-one#2978] [Questions] [answered]
Hi, I started to deploy my first nextcloud and I want to use with my existent shares from my synology (I look the docs about it) and one major feature is to use the current UID and GID, this image don't want to start support the custom UID and GID as [says here](https://github.com/nextcloud/all-in-one/discussions/1974), so I want the same pieces from AIO but without using it, because this is a deal break to me, there any docker compose that is close to emulate all feature that AIO has? I saw the docker compose from docker repository and this one, but seems only about database and reverse proxy. What I'm ask for is in docker compose with all [containers from AIO](https://github.com/nextcloud/all-in-one/tree/main/Containers) but in a descriptive file as a sample, there same in some place?

**Comment by szaimen** [comment, marked as answer]
Hi, why not mounting the shares from your synology into Nextcloud using the external storage app? See https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/external_storage_configuration_gui.html

---

> I saw the docker compose from docker repository and this one, but seems only about database and reverse proxy. What I'm ask for is in docker compose with all [containers from AIO](https://github.com/nextcloud/all-in-one/tree/main/Containers) but in a descriptive file as a sample, there same in some place?

You are probably looking for this? https://github.com/nextcloud/all-in-one/tree/main/manual-install

**Reply by webysther** [reply]
This file: https://github.com/nextcloud/all-in-one/blob/main/manual-install/latest.yml thanks know much easy to understand!

**Reply by webysther** [reply]
Maybe rename [latest.yml](https://github.com/nextcloud/all-in-one/blob/main/manual-install/latest.yml) to compose.yaml help someone.

**Reply by szaimen** [reply]
You could also read what is written there.
